default/scripting/buildings/EXPERIMENTOR_OUTPOST.focs.txt

```diff
@@ -78,15 +78,15 @@ BuildingType
                 Not OwnedBy affiliation = AnyEmpire
             ]
             activation = Or [
-                Turn high = [[EXPERIMENTOR_SPAWN_START_TURN]]      // always remove lanes before spawn start turn
+                Turn high = ([[EXPERIMENTOR_SPAWN_START_TURN]])      // always remove lanes before spawn start turn
```

Comment:
This seems to be doing the opposite of what you say in the summary...

Reply:
The same commit removes the parentheses from the macro itself so no nesting level is added here.
I wasn't completely sure if the parentheses are needed in places like this. If they aren't, they should be removed as well.

Reply:
I don't see why they would be... the macro should be a complete expression without them. If it parses and runs without error without them, then leave them off.

Reply:
In my tests, it doesn't parse properly without the brackets. Not sure why, but apparently they are needed.